src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -253,13 +253,13 @@ internal partial class BatchFixAllProvider : FixAllProvider
             return null;
         }
 
-        public virtual string GetFixAllTitle(FixAllState fixAllState)
+        private static string GetFixAllTitle(FixAllState fixAllState)
             => FixAllContextHelper.GetDefaultFixAllTitle(fixAllState.Scope, fixAllState.DiagnosticIds, fixAllState.Document, fixAllState.Project);
```

Comment:
virtuals that were not overridden were made non-virtual.  publics that were not accessed out of the tye were made private.